src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = new UserService().findByEmail(email);
```

Comment:
I would recommend to have a `UserService` as a class member. It will improve flexibility and simplify testing.

Reply:
I didn't quite understand the 2nd part, I hope I did as you asked